pkg/eventemitter/eventemitter.go

```diff
@@ -324,6 +331,14 @@ func (e *EventEmitter) emitEventByHandler(eventData eventdata.EventData) {
 
 	if eventData.HandlerKey == "" {
 		for key, handler := range e.eventHandlersCache {
+			// Filter Event
+			identifierKey := getPrefixIdentifierFromKey(key)
+
+			if e.eventFilterCache[identifierKey] != nil && !e.eventFilterCache[identifierKey].FilterEvent(eventData.CloudEventType) {
+				e.log.Info("Event is filtered", "cloudeventType", eventData.CloudEventType, "event identifier", identifierKey)
```

Comment:
This could be quite verbose, let's move it to other log level:
```suggestion
				e.log.V(1).Info("Event is filtered", "cloudeventType", eventData.CloudEventType, "event identifier", identifierKey)
```